tests/syscall/fs/enc/enc.cpp

```diff
@@ -775,6 +798,84 @@ void test_fs(const char* src_dir, const char* tmp_dir)
     test_dup_case2(tmp_dir);
 }
 
+void test_fs_linux(const char* src_dir, const char* tmp_dir)
```

Comment:
Should this be inside an ifdef if it's not needed on WIN32?

Reply:
edger8r does not yet support macro.

Reply:
Right, so I would recommend the API to be in a Linux-only EDL so it can be ifdef'ed out on WIN32.